apps/dc/dev/base/kustomization.yaml

```diff
@@ -6,3 +6,4 @@ resources:
 patches:
   #overlays for dc-purview-shir
   - path: ../../dc-purview-shir/dev.yaml
+  - path: ../../serviceaccount/stg.yaml
```

Comment:
Not sure if this is required, the docs advised to run a script that inserts this path